client/packages/android/app/src/main/AndroidManifest.xml

```diff
@@ -20,4 +21,7 @@
     <!-- Permissions -->
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.CAMERA" />
+
+    <uses-sdk tools:overrideLibrary="com.google.zxing.client.android" />
```

Comment:
How does this work?, e.g. where does the lib comes from if it isn't in the gradle file / how does it get fetched?